cmd/lhsm-plugin-az/main.go

```
@@ -94,8 +98,8 @@ func (a *archiveConfig) checkValid() error {
 }
 
 func (a *archiveConfig) checkAzAccess() error {
-	if _, err := azblob.NewSharedKeyCredential(a.AzStorageAccount, a.AzStorageKey); err != nil {
-		return errors.Wrap(err, "No Az credentials found; cannot initialize data mover")
+	if a.AzStorageKVName != "" && a.AzStorageKVSecretName != "" {
```

Shouldn't we return an error if there's no KV credentials?